subjects/printfirsthalf/README.md

```diff
@@ -0,0 +1,46 @@
+## printfirsthalf
+
+### Instructions
+
+Write a function called `PrintFirstHalf()` that takes a `string` as an argument and returns printed the first half of a `string`.
```

Comment:
grammarfix: (...) as an argument and prints the first half of a `string`.

Reply:
It is fixed